Okta.Xamarin/Okta.Xamarin.UITest.Shared/UITestData.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Okta.Xamarin;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Net.NetworkInformation;
+
+namespace Okta.Xamarin.UITest
+{
+	public class UITestData
+	{
+		public static UITestData Load(string filePath = "~/.okta/UITestData.json")
```

Comment:
Nit-pick: The indentation here looks different from other files. I think it'd be a good idea to use a linter (Stylecop for example) for consistency.